src/CrowdinApiClient/Api/Enterprise/NotificationApi.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace CrowdinApiClient\Api\Enterprise;
+
+use CrowdinApiClient\Api\AbstractApi;
+
+/**
+ * Send notifications
+ *
+ * @package Crowdin\Api\Entreprise
+ */
+class NotificationApi extends AbstractApi
+{
+
+    /**
+     * Send Notification To Organization Members
+     * @link https://developer.crowdin.com/enterprise/api/v2/#operation/api.notify.post  API Documentation Enterprise
+     *
+     * @param array $data
+     *  string $data[message] required<br>
+     *  string $data[role] Enum: "owner" "admin"<br>
+     *  integer[] $data[userIds]
+     * @return mixed
+     */
+    public function sendNotificationToOrganizationMembers(array $data)
+    {
+        $path = 'notify';
+
+        $options = [
+            'body' => json_encode($data),
+            'headers' => ['Content-Type' => 'application/json']
+        ];
+
+        return $this->client->apiRequest('post', $path, null, $options);
```

Comment:
Use the `_create` method

Reply:
`_create` method need a model name. The api return an 204 code with no data.